packages/gafl-webapp-service/src/pages/summary/licence-summary/route.js

```diff
@@ -113,12 +114,13 @@ class RowGenerator {
   }
 
   generateLicenceLengthRow () {
-    return this.generateStandardRow(
-      'licence_summary_length',
-      this.labels[`licence_type_${this.permission.licenceLength.toLowerCase()}`],
-      LICENCE_LENGTH.uri,
-      'change-licence-length'
-    )
+    const args = ['licence_summary_length', this.labels[`licence_type_${this.permission.licenceLength.toLowerCase()}`]]
+
+    if (!(this.permission.licenceType === mappings.LICENCE_TYPE['trout-and-coarse'] && this.permission.numberOfRods === '3')) {
```

Comment:
Given that there's no 3-rod salmon licence, could we just check the number of rods and ignore the licence type? Slightly simpler.